emqx/content.md

```diff
@@ -124,7 +115,7 @@ Let's create a static node list cluster from docker-compose.
 -	View cluster
 
 ```bash
-  $ docker exec -it my_emqx_emqx1_1 sh -c "emqx_ctl cluster status"
+  $ docker exec -it my_emqx_emqx1_1 sh -c "emqx ctl cluster status"
```

Comment:
For this one, maybe expand it to which version uses which command. i.e.

```bash
# Version x and below
$ docker exec -it my_emqx_emqx1_1 sh -c "emqx_ctl cluster status"

# Version and above
$ docker exec -it my_emqx_emqx1_1 sh -c "emqx ctl cluster status"

>  Cluster status: #{running_nodes => ['emqx@node1.emqx.io','emqx@node2.emqx.io'],
                    stopped_nodes => []}
```

Reply:
`emqx_ctl` is an alias for `emqx ctl`.
The former is a legacy one, was used in v4, all v5 images support both.